### PR TITLE
Support for collection member coercions issue #10

### PR DESCRIPTION
**Pull request by galetahub**
Also with some code style fixes

### CODE REVIEW

**Comment by galetahub** [comment]
Hi,
Any updates?

File: lib/fast_attributes.rb [hunk 1]
```diff
@@ -4,24 +4,24 @@
 require 'fast_attributes/version'
 require 'fast_attributes/builder'
 require 'fast_attributes/type_cast'
+require 'fast_attributes/type_casting'
 require 'fast_attributes/default_attributes'
 
 module FastAttributes
-  TRUE_VALUES  = {true => nil, 1 => nil, '1' => nil, 't' => nil, 'T' => nil, 'true' => nil, 'TRUE' => nil, 'on' => nil, 'ON' => nil}
-  FALSE_VALUES = {false => nil, 0 => nil, '0' => nil, 'f' => nil, 'F' => nil, 'false' => nil, 'FALSE' => nil, 'off' => nil, 'OFF' => nil}
+  TRUE_VALUES  = [true, 1, '1', 't', 'T', 'true', 'TRUE', 'on', 'ON'].freeze
```
Comment (miry) [inline lib/fast_attributes.rb]: @galetahub thanks for your work. it is impressive. Can you explain what the purpose of changing Map to Array? This approach was used for fast search. Here is my benchmarks: https://gist.github.com/miry/6c244b103ac87b2754f1bc696688fd7a

Thanks

Comment (galetahub) [reply lib/fast_attributes.rb]: Hm,
I saw that we store two hashes with nil values for store true/false values, so I decided to replace it with one array with true values.

But you are right hash method key? is faster then array include?
Maybe is it better to use one hash with true values?

Comment (miry) [reply lib/fast_attributes.rb]: @galetahub I think I got it what are you suggesting. And it should simplify. But can you post example what is your idea.

**Comment by danielvidal** [comment]
Hi folks, will this PR be released?
cc/ @galetahub @miry

**Comment by miry** [comment]
@danielvidal I suggest to create own fork with @galetahub 's changes. 
Also you would like to check https://github.com/intridea/hashie or https://github.com/solnic/virtus as alternative to fast_attribute.

**Pull request merged**